### ISSUE TITLE
Include all pages in conversion drawio -> ttl

### ISSUE THREAD

**Issue by GordianDziwis** [oeg-upm/Chowlk#37]
Currently, only the first page is exported, it could be useful to export all pages.

**Comment by mariapoveda** [comment]
But this is a draw.io requirement, right? Chowlk works with the export.

**Comment by GordianDziwis** [comment]
Yes the title was misleading, I mean that the rdf converter only picks up the first pages of the chowlk diagram.

**Comment by mariapoveda** [comment]
I see, but then one should ensure that each page is for a different ontology. Right now the worflow is supposed to be for one input and output ontology. 

We can think about this as future feature.

**Comment by GordianDziwis** [comment]
Why would you need to ensure, that they are all different ontologies?

**Comment by mariapoveda** [comment]
Because in my case I use tabs for keeping different versions of the same ontology, as it evolves, so differrent pages could be previous versions.

**Comment by danielhz** [comment]
I think it would be useful to offer the option to select a set of pages. The main benefit of this is that I can use some pages for the class diagrams, and other pages for instance examples.

For example, we could use the following command would print only pages 2 and 3:

```bash
python converter.py --pages 2,3 diagram.xml ontology.ttl --type ontology --format ttl
```

In the case we want all pages, we could use the following command:

```bash
python converter.py --all-pages diagram.xml ontology.ttl --type ontology --format ttl
```
